receiver/opencensusreceiver/config.go

```diff
@@ -32,6 +32,8 @@ type Config struct {
 	// TLSCredentials is a (cert_file, key_file) configuration.
 	TLSCredentials *tlsCredentials `mapstructure:"tls-credentials,omitempty"`
 
+	CorsOrigins []string `mapstructure:"cors-allowed-origins"`
```

Comment:
Please, refer to the original comment to describe this field on OpenCensus service:
```golang
	// The allowed CORS origins for HTTP/JSON requests the grpc-gateway adapter
	// for the OpenCensus receiver. See github.com/rs/cors
	// An empty list means that CORS is not enabled at all. A wildcard (*) can be
	// used to match any origin or one or more characters of an origin.
```